sqlmesh/core/macros.py

```diff
@@ -872,7 +872,7 @@ def star(
         for excluded in exclude.expressions or except_.expressions
     }
     quoted = quote_identifiers.this
-    table_identifier = alias.name or relation.name
+    table_identifier = normalize_identifiers(alias or relation, dialect=evaluator.dialect).name
```

Comment:
Trying this out real quick...

```suggestion
    table_identifier = normalize_identifiers(alias or relation, dialect=evaluator.dialect)
```